### PR TITLE
docs: redirect featureflagbook.com to best practices guide

### PR DESCRIPTION
**Pull request by ivarconr**
Should redirect "featureflagbook.com" to "https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices"

### CODE REVIEW

**Review by alvinometric — APPROVED** [review]
Neat, if everything else is setup on Vercel ha ha

File: website/vercel.json [hunk 1]
```diff
@@ -15,6 +15,17 @@
...
+    {
+      "source": "/",
+      "has": [
+        {
+          "type": "host",
+          "value": "www.featureflagbook.com"
+        }
+      ],
+      "destination": "https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices",
+      "permanent": true
```
Comment (ivarconr) [inline website/vercel.json:27]:
```suggestion
      "permanent": false
```

Comment (ivarconr) [reply website/vercel.json:27]: @alvinometric changing this to non-permanent in case the host matcher does not work how I hope it will.

**Comment by alvinometric** [comment]
Great, let's see how the preview looks

**Pull request merged**